src/models/multiChart.js

```diff
@@ -538,6 +538,7 @@ nv.models.multiChart = function() {
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
         showLegend: {get: function(){return showLegend;}, set: function(_){showLegend=_;}},
+        xScale: {get: function(){return x;}, set: function(_){ x = _; xAxis.scale(x); }},
```

Comment:
Oh wait, it looks like unittests failed...  sometimes (constantly) traviceCI bombs out though, if you run them yourself does it also fail?